src/Carousel.svelte

```diff
@@ -8,8 +8,8 @@
 	</div>
     {#if dots}
 	<ul>
-		{#each pips as pip, i}
-		<li on:click={() => go(i)} class={currentIndex === i ? "active" : ""}></li>
+		{#each Array(totalDots) as _, i}
+		<li test={i} on:click={() => go(i*perPage)} class={isDotActive(currentIndex, i) ? "active" : ""}></li>
```

Comment:
not sure you meant to leave `test` in there?

Reply:
certainly not haha. thanks!